stdlib_list/lists/3.10.txt

```diff
@@ -0,0 +1,1044 @@
+TK
```

Comment:
I'm not sure what's up with this `TK` module -- I don't see it in my local Python distribution.